ngraph/CMakeLists.txt

```diff
@@ -26,6 +26,14 @@ set(NGRAPH_INCLUDE_PATH
     ${CMAKE_CURRENT_SOURCE_DIR}/core/include
 )
 
+set(FRONTEND_INCLUDE_PATH
+    ${CMAKE_CURRENT_SOURCE_DIR}/frontend/generic/include
+)
+
+set(FRONTEND_BASE_PATH
```

Comment:
Looks like you don't use `FRONTEND_BASE_PATH`

Reply:
This will be useful for future frontends to include their own header files. Added a comment about this